### PR TITLE
PEP8 enabled

### PR DESCRIPTION
**Pull request by skupriienko**
## Links
* [DE-1038](https://mailgun.atlassian.net/browse/DE-1038)

### PEP-8

* Use double `"` quotes
* Use `autopep8` and `flake8` (with some additional plugins) as initial PEP8 linters
* Use `ruff` for linting and formatting (add `pyproject.toml` with the ruff configs)
* Sort imports by `ruff check --select I --fix` 
* Add `bandit` configs to `pyproject.toml` (no issues found by running `bandit -c pyproject.toml -r .`)
* ~~Fix the `exception`'s usage~~
* Apply linting rules:
  * [x] EXE001 Shebang is present but file is not executable
  * [x] UP009 [*] UTF-8 encoding declaration is unnecessary
  * [x] RET503 Missing explicit `return` at the end of function able to return non-`None` value
  * [x] UP004 [*] Class `Config` inherits from `object`
  * [x] SIM910 [*] Use `kwargs.get("version")` instead of `kwargs.get("version", None)`
  * [ ] _SKIP IT_: RET504 Unnecessary assignment to `response` before `return` statement
  * [x] B904 Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling
  * [x] UP032 [*] Use f-string instead of `format` call
  * [x] RUF010 [*] Use explicit conversion flag
  * [x] G002 Logging statement uses `%`
  * [x] S101 Use of `assert` detected 
  * [x] UP030 Use implicit references for positional format fields
  * [x] Q003 [*] Change outer quotes to avoid escaping inner quotes
  * [x] Use `Path(filename).read_text()` if you read the text file to a variable
  * [x] PTH100 `os.path.abspath()` should be replaced by `Path.resolve()`
  * [x] PTH120 `os.path.dirname()` should be replaced by `Path.parent`
  * [ ] _Skip temporarily_ TRY300 Consider moving response to an 'else' block; use secrets instead of random; add new linting and skipping rules
  * [x] UP015 [*] Unnecessary open mode parameters
  * [x] UP034 [*] Avoid extraneous parentheses
  * [x] S311 Standard pseudo-random generators are not suitable for cryptographic purposes: use secrets.choice()
  * [ ]  _Skip temporarily_ ARG001
  * [x] F401 imported but unused
  * [x] Fix E122, E125, E126, E131, E202, E501 (88 symbols)
  * [ ] _Skip_ the linting rule PLR0917 Too many positional arguments
  * [x] Fix E1101: Instance of 'Endpoint' has no '_doc' member (no-member)
  * [x] Fix C0325: Unnecessary parens after 'if' keyword

### CODE REVIEW

**Comment by skupriienko** [comment]
@DanyilNefodov Could you please run tests against these changes? Because I'm getting `KeyError: 'MJ_APIKEY_PUBLIC'` if I run `pytest test.py`

**Comment by skupriienko** [comment]
> @DanyilNefodov Could you please run tests against these changes? Because I'm getting `KeyError: 'MJ_APIKEY_PUBLIC'` if I run `pytest test.py`

I was able to run tests

**Comment by skupriienko** [comment]
I [reverted](https://github.com/mailjet/mailjet-apiv3-python/pull/107/commits/79eeedcfc50e0196e348ae7b6bfd22db2eb77bb3) `Client` error handling to see the whole traceback. Also, I [reverted](https://github.com/mailjet/mailjet-apiv3-python/pull/107/commits/bf8b3dd0217d6f4d2db059fef8b59f91a6aa17b5) `random` import. I think for tests, it's fine to use `random` instead of `secrets`.
And I solved the merge conflict.
@DanyilNefodov Could you please look at the PR?

File: samples/campaign_sample.py [hunk 1]
```diff
@@ -67,29 +61,25 @@ def send_the_campaign_right_away():
...
-                        "Name": "passenger 1"
-                    }
-                ],
+                        "Name": "passenger 1"}],
                 "Subject": "Your email flight plan!",
                 "TextPart": "Dear passenger 1, welcome to Mailjet! May the delivery force be with you!",
                 "HTMLPart": "<h3>Dear passenger 1, welcome to <a "
-                            "href=\"https://www.mailjet.com/\">Mailjet</a>!</h3><br />May the delivery force be with "
-                            "you!",
+                "href=\"https://www.mailjet.com/\">Mailjet</a>!</h3><br />May the delivery force be with "
```
Comment (DanyilNefodov) [inline samples/campaign_sample.py:78]: This one looks strange, previous version was more readable.

File: samples/contacts_sample.py [hunk 2]
```diff
@@ -74,41 +58,29 @@ def add_a_contact_to_a_contact_list():
...
+            {"Action": "unsub", "ListID": "987654321"},
         ]
     }
     return mailjet30.contact_managecontactslists.create(id=_id, data=data)
 
 
 def manage_multiple_contacts_in_a_list():
     """POST https://api.mailjet.com/v3/REST/contactslist/$list_ID
-                /managemanycontacts"""
+    /managemanycontacts"""
```
Comment (DanyilNefodov) [inline samples/contacts_sample.py:83]: I'm not sure if it's worth to split this URL, especially if it's shorter then 79 characters

**Review by DanyilNefodov — APPROVED** [review]
LGTM with nits:

File: samples/campaign_sample.py (hunk 1 above)
Comment (skupriienko) [reply samples/campaign_sample.py:78]: Thanks!

File: samples/contacts_sample.py (hunk 2 above)
Comment (skupriienko) [reply samples/contacts_sample.py:83]: Fair enough!

Comment (skupriienko) [reply samples/contacts_sample.py]: Fixed

File: samples/campaign_sample.py (hunk 1 above)
Comment (skupriienko) [reply samples/campaign_sample.py]: Fixed

**Pull request merged**